subsys/logging/Kconfig

```diff
@@ -297,7 +297,7 @@ config LOG_BACKEND_UART
 config LOG_BACKEND_RTT
 	bool "Enable Segger J-Link RTT backend"
 	depends on USE_SEGGER_RTT
-	default y
+	default y if !SHELL_BACKEND_RTT
```

Comment:
what if use will enable SHELL RTT backend?

Reply:
what happens in master now, that once user enables SHELL_BACKEND_RTT, LOG_BACKEND_RTT is also enabled and there output is doubled. This PR fixes it.